install/config.EXAMPLE.php

```diff
@@ -101,7 +101,9 @@
 const FONTS_PATH = ASSETS_PATH . 'fonts' . DS;
 
 const NEWS_XML_PATH = UP_PATH . 'xml' . DS . 'news.xml';
+const NEWS_V2_XML_PATH = UP_PATH . 'xml' . DS . 'news_v2.xml';
```

Comment:
tbh here I was thinking of naming the old paths to be `OLD_NEWS_XML_PATH`
and the new ones to be `NEWS_XML_PATH`
as v2 implies something changed inside the file.

And the online ones to be `online_news.xml` and `online_assets.xml`

Reply:
If I understood you correctly it should now be according to your comment